templates/documentation/sdks/security-best-practices.md

```diff
@@ -0,0 +1,55 @@
+# Security Best Practices
+
+Ensuring app security is paramount for developers for several compelling reasons:
+
+* User Trust: A secure app instills trust in users. When people know their data and privacy are protected, they are more likely to use the app and share personal information.
+
+* Reputation: Security breaches can severely damage a developer's reputation. News of vulnerabilities or data leaks can spread rapidly and deter users from using the app.
+
+* Legal and Compliance Issues: Failing to secure user data can lead to legal consequences, especially with the introduction of data protection regulations like GDPR and CCPA. Developers can face lawsuits, fines, or other penalties for non-compliance.
+
+* Financial Impact: Security breaches can be financially devastating. The costs associated with cleaning up a breach, compensating affected users, and lost business can be exorbitant.
+
+* Customer Retention: When users feel safe, they are more likely to stay loyal to the app. A secure app reduces churn and increases customer retention.
+
+* Competitive Advantage: Security can be a significant differentiator. Apps that prioritize security can gain a competitive edge, attracting more users and business partners.
+
+In summary, securing an app is not merely a technical necessity; it's a strategic imperative for developers to protect their users, reputation, and long-term success.
+
+## Securing your application
+
+We highly recommend to avoid storing the API keys in your frontend application as long as it is possible. The best way to make sure that your API keys are secure is separating the front-end and the backend, while making sure that all the API keys and sensitive app information is stored on the backend.
+
+* Make sure that the code that makes request to the API, including the API keys are stored on the backend. While the frontend is making requests to your backend securely. 
```

Comment:
Same sentence as the previous one? AI inside.

Reply:
good spotting, I just reordered the structure and missed it